upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -131,5 +131,22 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		manifests[key] = "addons/" + location
 	}
 
+	if b.cluster.Spec.Networking.Calico != nil {
+		key := "networking.calico"
```

Comment:
This technically should be a domain that you "own", so we don't get into naming problems.  (the weave one is not right..)

How about `networking.projectcalico.org`